src/main/scala/cli/package.scala

```diff
@@ -4,6 +4,9 @@ import parser.{ExtractedLogEntry, LogEntryWithRedoStatement}
 import schema.DatabaseSchema
 
 package object cli {
+
+  var strictPrimaryKeyChecking: Boolean = false
```

Comment:
I'm not sure whether we're abusing the companion object a bit in order to store that flag, but I found it preferable to the option of passing it all the way down to the column class where we need it. Also, the companion object is a singleton, which I think is more convenient. Or do you have a better idea, @tom-lichtenstein?